infrastructure/src/main/resources/corfu_replication_config_two_primaries.properties

```diff
@@ -0,0 +1,23 @@
+log_reader_max_retry=5
+log_reader_queue_size=5
+log_reader_resend_timer=100
+log_reader_error_on_message_timeout=true
+
+log_writer_queue_size=20
+log_writer_ack_cycle_count=2
+log_writer_ack_cycle_time=100
+
+primary_site="primary site"
```

Comment:
can we change all primary for active? to make it consistent throughout the code.. thanks!

Reply:
ok